### ISSUE TITLE
Warning in phar execution

### ISSUE THREAD

**Issue by Webysther** [squizlabs/PHP_CodeSniffer#1024]
After install in docker image:

```
RUN curl -OL https://squizlabs.github.io/PHP_CodeSniffer/phpcs.phar \
        && chmod +x phpcs.phar \
        && mv phpcs.phar /usr/local/bin/phpcs
```

I get this warning:

![captura de tela de 2016-05-28 00-32-38](https://cloud.githubusercontent.com/assets/750007/15624894/d9076600-246b-11e6-9584-d0405463c3b3.png)

My system configuration:

```
PHP 5.6.22 (cli) (built: May 28 2016 00:01:39) 
Copyright (c) 1997-2016 The PHP Group
Zend Engine v2.6.0, Copyright (c) 1998-2016 Zend Technologies
    with Zend OPcache v7.0.6-dev, Copyright (c) 1999-2016, by Zend Technologies
    with Xdebug v2.4.0, Copyright (c) 2002-2016, by Derick Rethans


composer-plugin-api 1.1.0    The Composer Plugin API
ext-bcmath          0        The bcmath PHP extension (actual version: )
ext-bz2             0        The bz2 PHP extension (actual version: )
ext-calendar        0        The calendar PHP extension (actual version: )
ext-ctype           0        The ctype PHP extension (actual version: )
ext-curl            0        The curl PHP extension (actual version: )
ext-date            5.6.22   The date PHP extension
ext-dba             0        The dba PHP extension (actual version: )
ext-dom             20031129 The dom PHP extension
ext-ereg            0        The ereg PHP extension (actual version: )
ext-exif            0        The exif PHP extension (actual version: 1.4 $Id: db007ca2e6d0b4513ae77990972997fad8bfe9c9 $)
ext-fileinfo        1.0.5    The fileinfo PHP extension
ext-filter          0.11.0   The filter PHP extension
ext-ftp             0        The ftp PHP extension (actual version: )
ext-gd              0        The gd PHP extension (actual version: )
ext-gettext         0        The gettext PHP extension (actual version: )
ext-hash            1.0      The hash PHP extension
ext-iconv           0        The iconv PHP extension (actual version: )
ext-intl            1.1.0    The intl PHP extension
ext-json            1.2.1    The json PHP extension
ext-libxml          0        The libxml PHP extension (actual version: )
ext-mbstring        0        The mbstring PHP extension (actual version: )
ext-mcrypt          0        The mcrypt PHP extension (actual version: )
ext-memcached       2.2.0    The memcached PHP extension
ext-mysqli          0.1      The mysqli PHP extension
ext-mysqlnd         0        The mysqlnd PHP extension (actual version: mysqlnd 5.0.11-dev - 20120503 - $Id: 76b08b24596e12d4553bd41fc93cccd5bac2fe7a $)
ext-openssl         0        The openssl PHP extension (actual version: )
ext-pcre            0        The pcre PHP extension (actual version: )
ext-PDO             1.0.4dev The PDO PHP extension
ext-pdo_mysql       1.0.2    The pdo_mysql PHP extension
ext-pdo_sqlite      1.0.1    The pdo_sqlite PHP extension
ext-Phar            2.0.2    The Phar PHP extension
ext-posix           0        The posix PHP extension (actual version: )
ext-readline        5.6.22   The readline PHP extension
ext-Reflection      0        The Reflection PHP extension (actual version: $Id: fbcf7a77ca8e3d4cd7501de8025235b947b8240f $)
ext-session         0        The session PHP extension (actual version: )
ext-shmop           0        The shmop PHP extension (actual version: )
ext-SimpleXML       0.1      The SimpleXML PHP extension
ext-soap            0        The soap PHP extension (actual version: )
ext-sockets         0        The sockets PHP extension (actual version: )
ext-SPL             0.2      The SPL PHP extension
ext-sqlite3         0.7-dev  The sqlite3 PHP extension
ext-sysvmsg         0        The sysvmsg PHP extension (actual version: )
ext-sysvsem         0        The sysvsem PHP extension (actual version: )
ext-sysvshm         0        The sysvshm PHP extension (actual version: )
ext-tokenizer       0.1      The tokenizer PHP extension
ext-wddx            0        The wddx PHP extension (actual version: )
ext-xml             0        The xml PHP extension (actual version: )
ext-xmlreader       0.1      The xmlreader PHP extension
ext-xmlrpc          0.51     The xmlrpc PHP extension
ext-xmlwriter       0.1      The xmlwriter PHP extension
ext-Zend-OPcache    7.0.6    The Zend OPcache PHP extension (actual version: 7.0.6-devFE)
ext-zip             1.12.5   The zip PHP extension
ext-zlib            2.0      The zlib PHP extension
lib-curl            7.38.0   The curl PHP library
lib-iconv           2.19     The iconv PHP library
lib-ICU             52.1     The intl PHP library
lib-libxml          2.9.1    The libxml PHP library
lib-openssl         1.0.1.11 OpenSSL 1.0.1k 8 Jan 2015
lib-pcre            8.38     The pcre PHP library
php                 5.6.22   The PHP interpreter
php-64bit           5.6.22   The PHP interpreter, 64bit
```

**Comment by gsherwood** [comment]
Duplicate of #944

Fixed in master. Will be released in 2.6.1,  when I can finally get some time.
